packages/button/src/react/index.js

```diff
@@ -59,7 +59,7 @@ const styleAppearance = ({ appearance }) =>
       background: 'none',
       ':hover': {
         color: core.colors.white,
-        background: core.colors.gray04
+        background: 'rgba(255, 255, 255, 0.15)'
```

Comment:
@dborodyansky , Do you want to base this on the hard-coded rgb values, or would you rather base this on `core.colors.white`?  If the later, that's possible.  See the card's usage of:

```
import { transparentize } from 'polished'
```

If the former, let's merge this.

Reply:
The former is preferred, just didn't know it was possible :) let me give it a go.